services/containers/service.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 func init() {
-	plugin.Register("containers-grpc", &plugin.Registration{
+	plugin.Register(&plugin.Registration{
 		Type: plugin.GRPCPlugin,
+		ID:   "io.containerd.containers-grpc",
```

Comment:
So we could do something with the `ID: "containers"` and concat the type + id to make it
`io.containerd.grpc.v1.containers` in the registration?